components/ProfileSettings/ProfileSettings.module.css

```diff
@@ -0,0 +1,12 @@
+.button {
+  width: 100px;
+  height: 31px;
+  background-color: #6272aa;
```

Comment:
use the colors from the global styles so it would be `--var(COLORNAME)` If the color does not exist then you can create it too. Do this for all the times you hardcode a hex code